apps/gro/controllers/address-start.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const BaseController = require('hof').controllers.base;
+const Client = require('node-rest-client').Client;
+const config = require('../../../config');
+
+module.exports = class AddressStartController extends BaseController {
+  constructor(options) {
+    super(options);
+  }
+
+  supercall() {
+    super.successHandler.apply(this, arguments);
+  }
```

Comment:
what's the reasoning for saving a reference to this super method? Cant we just call it where it is needed?